python/ray/tune/schedulers/pbt.py

```diff
@@ -326,11 +355,11 @@ def __init__(
                     "tune.uniform, tune.loguniform, etc."
                 )
 
-            if not hyperparam_mutations and not custom_explore_fn:
-                raise TuneError(
-                    "You must specify at least one of `hyperparam_mutations` "
-                    "or `custom_explore_fn` to use PBT."
-                )
+        if not hyperparam_mutations and not custom_explore_fn:
```

Comment:
This was a useless check previously, since the indentation was within the loop and making it into the loop means that `hyperparam_mutations` is never empty.